scripts/funcotator/testing/testFuncotator.sh

```diff
@@ -301,8 +301,8 @@ if [[ $r -eq 0 ]] && ${doRunLargeTests} ; then
   $useAOUDataSources && echo "Using AOU data sources." && DATA_SOURCES_PATH=/Users/jonn/Development/funcotator_dataSources.vAoU3
 
   # Use cloud data sources if we need them:
-  $useCloudDataSources && echo "Using cloud data sources." && DATA_SOURCES_PATH=/Users/jonn/Development/gatk/src/test/resources/large/funcotator/funcotator_dataSources_cloud_gnomad/
-  #$useCloudDataSources && echo "Using cloud data sources." && DATA_SOURCES_PATH=gs://hellbender/test/resources/large/funcotatorDataSourceCollection/funcotator_dataSources_cloud/
+  #$useCloudDataSources && echo "Using cloud data sources." && DATA_SOURCES_PATH=/Users/jonn/Development/gatk/src/test/resources/large/funcotator/funcotator_dataSources_cloud_gnomad/
+  $useCloudDataSources && echo "Using cloud data sources." && DATA_SOURCES_PATH=gs://broad-dsde-methods-jonn/funcotator_dataSources.v1.5.20181119s
```

Comment:
You should move this file somewhere accessible to the public, right?  This bucket is private.

Reply:
No.  This is a testing script that I'm using to test Funcotator internally.  Nothing in this file is guaranteed to work for anyone without some editing.  It is checked in so that it will not be lost.